whatshap/offspringscoring.py

```diff
@@ -0,0 +1,279 @@
+import logging
```

Comment:
What do you mean with module-level? Once per module (like `phase`, `polyphase`, etc.) or once per (reasonably big) file?

Reply:
I meant a triple-quoted string at the very beginning of the file that explains why the file exists, somewhat like this one:

https://github.com/whatshap/whatshap/blob/43af40d20eb66f0b8bc3018fb786cf6fe3e0f6cc/whatshap/pedigree.py#L1-L3

This isn’t such a good example (because it essentially just restates the file name), but perhaps one sentence about what this offspring scoring is or does would be nice.